frog/imports/client/AccountModal/SignUp.js

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
```

Comment:
This should not be here. `CssBaseline` should only be used once at the root of FROG.

Reply:
Fixed